controllers/v1/solutionsController.js

```diff
@@ -1888,18 +1700,52 @@ module.exports = class Solutions extends Abstract {
    * @param {String} req.params._id observation solution link.
    * @returns {JSON} observation data.
    */
+  /**
+   * verify Link
+   * @method
+   * @name verifyLink
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - solution link
+   * @returns {Array}
+   */
 
   async verifyLink(req) {
     return new Promise(async (resolve, reject) => {
       try {
-        let result = await observationsHelper.verifyLink(
+        let solutionData = await solutionsHelper.verifyLink(
           req.params._id,
-          req.rspObj.userToken,
-          req.userDetails.userId,
           req.body,
+          req.userDetails.userId,
+          req.userDetails.userToken,
+          req.query.hasOwnProperty('createProject') ? gen.utils.convertStringToBoolean(req.query.createProject) : true,
```

Comment:
@praveenKDass create project query will not come here right? hence we can set it as true by default

Reply:
Setting as true by default @VISHNUDAS-tunerlabs